skimage/filters/thresholding.py

```diff
@@ -124,11 +124,11 @@ def thresh(func):
         """
         Select a wrapper function to return a thresholded image. 
         """
-        if ( func.__name__ == 'threshold_local' ):
+        if func == threshold_local:
```

Comment:
```suggestion
        if func is threshold_local:
```
We want to test for the identity of the function, not for its value. A semantic but in my opinion good distinction to make. :wink:

Reply:
Oh, I didn't know the operator. I am learning a lot more than anticipated =)